docs/docs/api/promise.off.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: api
+id: promise.off
+title: Promise.off
+---
+
+
+[← Back To API Reference](/docs/api-reference.html)
+<div class="api-code-section"><markdown>
+##Promise.off
+
+```js
+Promise.off(String eventName("created"|"chained"|"fulfilled"|"rejected"|"following"|"cancelled"), Function handler) -> undefined
+```
+Removes a handler form promise lifecycle event (see: [on](promise.on.html).
+Given event name and handler function that were used for subscription, removes the subscription.
```

Comment:
What about something similar to what jQuery has in the docs? `The .off() method removes event handlers that were attached with .on().` and shortly explain the params?